### PR TITLE
Feature/heading tags

### PR DESCRIPTION
**Pull request by meirege**
Fixes link styling inside heading tags: #22

### CODE REVIEW

File: components/00-settings/_vars.scss [hunk 1]
```diff
@@ -64,3 +64,34 @@ $radius-4: 30px !default;
 
 // Breadcrumb settings.
 $bg-hover-breadcrumb: $color-lightest-blue !default;
+
+// Headings.
+$heading-1-color: $color-darkest-blue;
```
Comment (Jeroen005) [inline components/00-settings/_vars.scss]: If we add those settings to the _vars.scss would it be clear for other developers that even if it's overwritten in the settings file the section will have more priority?

Comment (meirege) [inline components/00-settings/_vars.scss]: Probably not at first... so I propose we add a separate readme in the docs of the style guide that explains sections and why they override things like these variables.
Also, maybe we can add some sort of comment in the _vars file that tells developers that these variables may be subject to change due to the use of sections?

Comment (Jeroen005) [inline components/00-settings/_vars.scss]: Sounds like a good idea.

Comment (meirege) [inline components/00-settings/_vars.scss]: I've added the doc page and comments in _vars.scss

Comment (wim-vantomme) [inline components/00-settings/_vars.scss]: Ok for me

**Pull request merged**